utils/utils.go

```diff
@@ -46,3 +47,28 @@ func GetStringFromEnvWithDefault(enVarName, defaultValue string) string {
 
 	return envValue
 }
+
+//MustGetStringFromEnv panics if OS Environment Variable is not set
+func MustGetStringFromEnv(enVarName string) string {
+	envValue := os.Getenv(enVarName)
+	if envValue == "" {
+		panic(fmt.Sprintf("%s must not be empty", enVarName))
+	}
+
+	return envValue
+}
+
+// GetBoolFromEnvWithDefault parses an OS Environment Variable as bool
+func GetBoolFromEnvWithDefault(enVarName string, defaultValue bool) (bool, error) {
+	envValue := os.Getenv(enVarName)
+	if envValue == "" {
+		return defaultValue, nil
+	}
+
+	envBool, err := strconv.ParseBool(envValue)
+	if err != nil {
+		return defaultValue, err
```

Comment:
FYI: Feel free to just panic here: 
We expect this to only fail, if there is a typo in the config which is panic worthy in every case I can imagine. (if you panic, decorate the `err` with a bit of context - i.e. something like `fmt.Errorf("couldn't parse %s as bool: %s", enVarName, err)`)